tests/Microsoft.Data.DataFrame.Tests/DataFrameTableTests.cs

```diff
@@ -0,0 +1,128 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Collections.Generic;
+using Microsoft.Data;
+using Xunit;
+
+namespace Microsoft.Data.Tests
+{
+    public class DataFrameTableTests
```

Comment:
This should just be `DataFrameTests` right?  `DataFrameTable` is an internal type and we generally write unit tests at the public API surface.

Reply:
Done